libursa/wasm/gen.pl

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/perl
```

Comment:
Perl? Why not have this written in rust?

Reply:
File doesn't exist.